micromamba/tests/test_create.py

```diff
@@ -652,6 +652,24 @@ def test_spec_with_channel_and_subdir():
         )
 
 
+def test_spec_with_slash_in_channel(tmp_home, tmp_root_prefix):
+    env_name = "myenv"
+    try:
```

Comment:
prefer [`with pytest.raises`](https://docs.pytest.org/en/stable/reference/reference.html#pytest.raises) for negative testing. If the create command succeed, the test will still pass.